functions/data/users.dataset.js

```diff
@@ -35,4 +32,19 @@ module.exports = [
         team: generateTeam(),
         uid: falso.randUuid(),
     },
-]
+    {
```

Comment:
Should create a function named `generateUser` and just export an array `[generateUser(), generateUser()]`

Check if the 2 users are different, if not, check about `seed` from falso